### PR TITLE
WDS-61 - Alert component

### PR DESCRIPTION
**Pull request by 0x009922**
- New component - `SAlert`. It is very similar to `SNotificationBody`
- `NotificationType` enum is renamed to `Status` and moved to general types section. `type` prop renamed to `status`

```vue
<template>
  <SAlert
    title="Your application is pending"
    status="info"
  />
</template>
```

### CODE REVIEW

File: packages/ui/.storybook/main.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function resolve(...paths) {
 }
 
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
Comment (stefashkaa) [inline packages/ui/.storybook/main.js:9]: why did you remove `mdx`? I guess we'll need it

**Review by stefashkaa — APPROVED** [review]

**Review by alexnatalia — APPROVED** [review]
